### PR TITLE
Fixing issue in SetAddStmt

### PR DESCRIPTION
**Pull request by johanfylling**
where `nil` values were equated to a missing value

Signed-off-by: Johan Fylling <johan.dev@fylling.se>

### CODE REVIEW

File: src/test/clojure/jarl/eval_test.clj [hunk 1]
```diff
@@ -130,7 +131,8 @@
           state {:local {}}
           state (set-local state target existing-value)
           state (set-local state 3 value)]
-      (is (thrown-with-msg? Exception #"complete rules must not produce multiple outputs" (eval-AssignVarOnceStmt source-index target state)))))
+      (is (thrown-with-msg? Exception (re-pattern "<\\{\"type\" \"local\", \"value\" 3}> already assigned")
+                            (eval-AssignVarOnceStmt source-index target state)))))
```
Comment (johanfylling) [inline src/test/clojure/jarl/eval_test.clj:135]: Drive-by fix

**Review by anderseknert — APPROVED** [review]
Nice!
